.github/workflows/publish-swift.yml

```diff
@@ -0,0 +1,134 @@
+          cp ../target/darwin-universal/release/libbreez_sdk_bindings.a bindings-swift/breez_sdkFFI.xcframework/macos-arm64_x86_64/breez_sdkFFI.framework/breez_sdkFFI
+          rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.h
+          rm bindings-swift/Sources/BreezSDK/breez_sdkFFI.modulemap
+
+      - name: Compress xcframework
+        working-directory: breez-sdk/libs/sdk-bindings/bindings-swift
+        run: |
+          zip -9 -r breez_sdkFFI.xcframework.zip breez_sdkFFI.xcframework
+          echo "XCF_CHECKSUM=`swift package compute-checksum breez_sdkFFI.xcframework.zip`" >> $GITHUB_ENV
+
+      - name: Update swift package
+        working-directory: breez-sdk/libs/sdk-bindings/bindings-swift
+        run: |
+          # Update package definition
+          sed 's#.binaryTarget(name: "breez_sdkFFI", path: "./breez_sdkFFI.xcframework"),#.binaryTarget(name: "breez_sdkFFI", url: "https://github.com/breez/breez-sdk-swift/releases/download/${{ inputs.package-version }}/breez_sdkFFI.xcframework.zip", checksum: "${{ env.XCF_CHECKSUM }}"),#;/.testTarget(name: "BreezSDKTests", dependencies: \["BreezSDK"\]),/d' Package.swift > ../../../../breez-sdk-swift/Package.swift
+          # Update language bindings
+          cp -r Sources ../../../../breez-sdk-swift
+
+      - name: Update cocoapods definitions
+        working-directory: breez-sdk-swift
+        run: |
+          sed -i '' 's#^.\{2\}spec.version.*$#  spec.version                = "${{ inputs.package-version }}"#' breez_sdkFFI.podspec
+          sed -i '' 's#^.\{2\}spec.version.*$#  spec.version                = "${{ inputs.package-version }}"#' BreezSDK.podspec
+
+      - name: Tag swift package
+        working-directory: breez-sdk-swift
+        if: ${{ inputs.publish }}
+        run: |
+          git add Package.swift
+          git add Sources
+          git add breez_sdkFFI.podspec
+          git add BreezSDK.podspec
+          git commit -m "Update Breez SDK Swift bindings to version ${{ inputs.package-version }}"
+          git push
+          git tag ${{ inputs.package-version }} -m "${{ inputs.package-version }}"
+          git push --tags
+
+      - name: Release and attach XCFramework binary artifact
+        if: ${{ inputs.publish }}
+        uses: ncipollo/release-action@v1
+        with:
+          repo: breez/breez-sdk-swift
+          tag: ${{ inputs.package-version }}
+          token: ${{ secrets.GITHUB_TOKEN_BREEZ_SDK_SWIFT }}
+          name: ${{ inputs.package-version }}
+          artifacts: "breez-sdk/libs/sdk-bindings/bindings-swift/breez_sdkFFI.xcframework.zip"
+          prerelease: true
+
+      - name: Push update to Cocoapods trunk
+        working-directory: breez-sdk-swift
+        if: ${{ inputs.publish }}
+        env:
+          COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
+        run: |
+          pod trunk push breez_sdkFFI.podspec --allow-warnings
+          pod trunk push BreezSDK.podspec --allow-warnings --synchronous
```

Comment:
It would be nice to upload an artifact in this workflow, like we do for other packages, like [flutter](https://github.com/breez/breez-sdk-greenlight/blob/main/.github/workflows/publish-flutter.yml#L86-L92), [react-native](https://github.com/breez/breez-sdk-greenlight/blob/main/.github/workflows/publish-react-native.yml#L56-L60), [android](https://github.com/breez/breez-sdk-greenlight/blob/main/.github/workflows/publish-android.yml#L56-L60), etc. That way we can use that in the breez-sdk-docs to build docs independent of published versions.